### DISCUSSION TITLE
remark-parse doesn't parse **abc(abc)** in paragraph/sentence

### DISCUSSION THREAD

**Discussion by CreamNuts** [remarkjs/remark#602] [Q&A] [answered]
I use remark-parse below
``` javascript
const content = "I want to use this **AWS(Amazon Web Services)**2"
const processedContent = await unified()
    .use(markdown) //remark-parse
    .use(remark2rehype) //remark-rehype
    .use(rehype2string) //rehype-stringify
    .process(content);
```

I expect 
> I want to use this **AWS(Amazon Web Services)**(no space)2

but actual output is
> I want to use this **AWS(Amazon Web Services)**2

After realizing that it was due to gfm, I tried many things but I didin't succeed. The example is English, but the actual language used is different, so it is important to solve this. How can I do that?

**Comment by wooorm** [comment, marked as answer]
This actual behavior conforms to the spec: https://spec.commonmark.org/dingus/?text=I%20want%20to%20use%20this%20**AWS(Amazon%20Web%20Services)**2

**Reply by ChristianMurphy** [reply]
To add on, an alternative would be to use a zero width spacing character, like a word joiner.
```markdown
I want to use this **AWS(Amazon Web Services)**&#8288;2
```
https://astexplorer.net/#/gist/91a7508b77b1836eef5b16cffe5f1c74/cf69817521ad9fc1b4d29cb305be4631f9a9ae3f

**Reply by CreamNuts** [reply]
I realized that is actual behavior but I want to ues. So @ChristianMurphy's alternative works well! Thanks for @wooorm , @ChristianMurphy !
